resources/dicts/patrols/general/border.json

```diff
@@ -2176,5 +2176,298 @@
             }
         ]
 
-    }
-]
\ No newline at end of file
+    },
+{
+    "patrol_id": "gen_bord_appmission",
```

Comment:
since this patrol feels a bit "adversarial", it might be good to lock it behind a certain amount of dislike between mentor and app.  it would be a little odd to get the outcome where the mentor calls their app irritating, only to check their relationships and see they like each other.

Reply:
I’ve changed the dislike requirement to be 10 for now, but lmk if you think it should be higher!